index.html

```diff
@@ -0,0 +1,131 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>togoville jazz festival homepage</title>
+</head>
+<body>
+     
+    <header>
+        <ul class="header-links">
+            <li><a href="#" class="social-link fa fa-twitter"></a></li>
+            <li><a href="#" class="social-link fa fa-facebook"></a></li>
+            <li><a class="lato-font-black" href="#">English</a></li>
+            <li><a class="lato-font-black" href="#">My page</a></li>
+            <li><a class="lato-font-black" href="#">Logout</a></li>
+        </ul>
+    </header>
+    
+    <main> 
+        <nav class="nav-bar">
+            <div class="hamburger-class">
+                <a href="#" ><img src="./img/iconmonstr-menu-1-240.png" alt="menu button"></a>
+            </div>
+            <ul class="menu-bar">
+                <li class="menu-logo" ><a href="./index.html"><img src="./img/level production icon 3.png" alt="level production icon"></a></li>
+                <li><a class="lato-font-black aboutme-link" href="./aboutme.html">About</a></li>
+                <li><a class="lato-font-black" href="#program">Program</a></li>
+                <li><a class="lato-font-black" href="#">Join</a></li>
+                <li><a class="lato-font-black" href="#sponsor">Sponsor</a></li>
+                <li><a class="lato-font-black" href="#">News</a></li>
+                <li class="campaign"><a class="lato-font-red" href="#">LP campaign</a></li>
+            </ul>
+        </nav> 
+        
+        <section class="headline">
+            <p class="lato-font-red" id="greeting">Hello! Music lovers</p>
+            <h1 class="lato-font-red">TOGOVILLE JAZZ <br> JAZZ FESTIVAL <br> 2022</h1>
+            <p class="lato-font-black" id="introduction">A lively festival celebrating popularizing, sharing of musical knowledge and discussing the political and cultural aspects that emerge from it</p>
+            <p class="lato-font-black" id="event-date">2022.05.15(TUE) ~ 27(SUN)</p>
+            <p class="lato-font-black">@French Institute of Togo, Goethe Institute,  and more</p>
+
+        </section>
+
+        <section class="main-program" id="program">
+            <h2 class="lato-font-red">Main Program<span class="underline-style"></span></h2>
+            <ul>
+                <li>
+                    <a href="#">
+                        <div>
+                            <img class="program-content" src="./img/root/debate.png" alt="debate icon">
+                            <h3 class="program-centent lato-font-red">Debate</h3>
+                            <p class="program-content lato-font-black">learn from our special guests and interact by answering and asking questions</p>
+                        </div>
```

Comment:
- [ ] Since this part is a self-contained entity on the page, I think it would be better to use the semantic `article` tag instead of a `div`. Kindly do the same for all 5 cards.  🧐️ You can check this [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) to get more insight on when to use the `article` element.
- [ ] Please update the `font-size` of the description paragraph. Currently, the `font-size` on the title and description are the same.